design/Bootstrap_design.Rmd

```diff
@@ -94,7 +94,7 @@ From the bootstrap_HR() function we will export the HR as a numeric value.
 
 ### Example code 
 
-```{r}
+```{r}`r ''`
```

Comment:
@MikeJSeo Added this so that the file can be knitted as I had entered example code as text.